### PR TITLE
Added missing option

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/7155

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @e0i, for your information

Thank you

File: windows/security/threat-protection/security-policy-settings/interactive-logon-display-user-information-when-the-session-is-locked.md [hunk 1]
```diff
@@ -61,6 +61,11 @@ This setting has these possible values:
     This change makes this setting consistent with the functionality of the new **Privacy** setting.
     To display no user information, enable the Group Policy setting **Interactive logon: Don't display last signed-in**.
 
+-   **Domain and user names only**
+
+    For a domain logon only, the domain\username is displayed.
+    The **Privacy** setting is automatically on and grayed out.
+    
 -   Blank.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/security-policy-settings/interactive-logon-display-user-information-when-the-session-is-locked.md:69]:
```suggestion
-   **Blank**
```

Comment (JohanFreelancer9) [reply windows/security/threat-protection/security-policy-settings/interactive-logon-display-user-information-when-the-session-is-locked.md:69]: Additional edit, please.

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**